nncf/onnx/graph/node_utils.py

```diff
@@ -0,0 +1,49 @@
+"""
+ Copyright (c) 2023 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import onnx
+import numpy as np
+
+from nncf.common.graph.graph import NNCFNode
+from nncf.onnx.graph.metatypes.onnx_metatypes import LAYERS_WITH_BIAS_METATYPES
+from nncf.onnx.graph.onnx_graph import ONNXGraph
+from nncf.onnx.graph.metatypes.onnx_metatypes import ONNX_OPERATION_METATYPES
+from nncf.onnx.graph.metatypes.onnx_metatypes import ONNXIdentityMetatype
+
+
+def is_node_with_bias(node: NNCFNode) -> bool:
+    """
+    :param node:
+    :return:
```

Comment:
Please don't forget to add a description.

Reply:
Done